### PR TITLE
[Fix] Calculation of number of NaNs in signal_fixpeaks()

### PR DESCRIPTION
**Pull request by danibene**
# Description

This PR aims to improve the previous PR: https://github.com/neuropsychology/NeuroKit/pull/651
See also the related issue: https://github.com/neuropsychology/NeuroKit/issues/650

# Proposed Changes

I changed the `neurokit` method of the `signal_fixpeaks()` function so that the number of NaNs inserted is computed as follows:
1. Rounding rather than taking the floor when getting an integer value for the number of NaNs (calculated by dividing large gaps by the mean interval)
2. Calculating the mean interval based on the original interval values and not the standardized interval values in the case that `relative_interval_max` is given.

# Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/neuropsychology/NeuroKit/blob/master/.github/CONTRIBUTING.rst#structure-and-code) file.
- [x] My PR is targeted at the **dev branch** (and not towards the master branch).
- [x] I ran the [CODE CHECKS](https://github.com/neuropsychology/NeuroKit/blob/master/.github/CONTRIBUTING.rst#run-code-checks) on the files I added or modified and fixed the errors.
  I got the following error, but I'm just going to ignore it if that's okay, since I don't think I changed anything that should cause this? 
  `astroid.exceptions.TooManyLevelsError: Relative import with too many levels (2) for module 'signal_fixpeaks'` Could also be the fact that I have a file called `signal_fixpeaks.py` in a different folder.
- [ ] I have added the newly added features to **News.rst** (if applicable)

### CODE REVIEW

**Comment by DominiqueMakowski** [comment]
Minor style issue https://github.com/neuropsychology/NeuroKit/runs/7420854905?check_suite_focus=true#step:8:23 to fix before we can merge!

**Comment by danibene** [comment]
> Minor style issue https://github.com/neuropsychology/NeuroKit/runs/7420854905?check_suite_focus=true#step:8:23 to fix before we can merge!

@DominiqueMakowski if I understood correctly it's an issue with signal_detrend()? Should I modify that file?

**Comment by DominiqueMakowski** [comment]
oops yeah my bad I introduced that just now 🙈 I'll fix it on dev then

File: AUTHORS.rst [hunk 1]
```diff
@@ -29,7 +29,7 @@ Core contributors
 Contributors
 -------------
 
-
+* `Danielle Benesch <https://github.com/danibene>`_ *(École de technologie supérieure, Canada)*
```
Comment (danibene) [inline AUTHORS.rst:32]: 🤩

**Comment by DominiqueMakowski** [comment]
@danibene apologies for the delay - it's been a busy month! I'm gonna merge this once it passes the checks :)

**Comment by danibene** [comment]
@DominiqueMakowski nice, thanks :)

**Pull request merged**